app/assets/javascripts/discourse/app/lib/sidebar/user/categories-section/category-section-link.js

```diff
@@ -169,8 +169,12 @@ export default class CategorySectionLink {
     return "span";
   }
 
-  get prefixElementColors() {
-    return [this.category.parentCategory?.color, this.category.color];
+  get prefixValue() {
+    if (this.category.parentCategory?.color) {
+      return [this.category.parentCategory?.color, this.category.color];
+    } else {
+      return [this.category.color, this.category.color];
```

Comment:
Related to my other comment, this could just end up as `[this.category.color]`

Reply:
I've simplified this to just passing `[this.category.color]`